Integrations/integration-MaxMind_GeoIP2.yml

```diff
@@ -0,0 +1,293 @@
+description: Enriches IP addresses
+detaileddescription: 'The MaxMind GeoIP2 integration allows you to query the MaxMind
+  API service and retrieve a JSON of all details.'
+configuration:
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Account ID
+  name: account
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Use system proxy
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Service Version
+  name: mode
+  defaultvalue: city
```

Comment:
Forgot to say that we should probably state the options available

Reply:
In documentation yes?